base/applications/notepad/dialog.c

```diff
@@ -220,13 +220,7 @@ static void AlertPrintError(void)
  */
 BOOL FileExists(LPCTSTR szFilename)
 {
-    WIN32_FIND_DATA entry;
-    HANDLE hFile;
-
-    hFile = FindFirstFile(szFilename, &entry);
-    FindClose(hFile);
-
-    return (hFile != INVALID_HANDLE_VALUE);
+    return GetFileAttributes(szFilename) != INVALID_FILE_ATTRIBUTES;
```

Comment:
Does this behave the same with a directory?

Reply:
On the case of a directory, the function will `TRUE` in the current PR.

Reply:
Samely.

Reply:
But there are some exceptions. `FindFirstFile` function accepts wildcard patterns.